docs/Reference/API-Methods.md

```diff
@@ -2459,6 +2459,16 @@ This can only be performed for the master organization and requires the majority
 
 ## Privacy methods
 
+!!! note
+
+    The following privacy methods have been created or modified in addition to the standard Ethereum `eth` JSON-RPC
```

Comment:
is this a standard messaging we use? as is not really a new method or modified. 
it's mainly making sure the `gasPrice` argument that already exists in standard Ethereum is sent with 0.

@ConsenSys/protocol-go-quorum what do you think?

Reply:
Slightly edited this note.